po/cs.po

```diff
@@ -0,0 +1,268 @@
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2019-10-24 01:00+0530\n"
+"PO-Revision-Date: 2021-08-09 17:52+0200\n"
+"Last-Translator: Vojtěch Perník <translations@pervoj.cz>\n"
+"Language-Team: Czech <gnome-cs-list@gnome.org>\n"
+"Language: cs\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=3; plural=(n==1) ? 0 : (n>=2 && n<=4) ? 1 : 2\n"
+"X-Generator: Gtranslator 40.0\n"
+
+#: src/Application.vala:156
+msgid "Blink your eyes"
+msgstr "Mrkněte očima"
+
+#: src/Application.vala:157
+msgid "Look away from the screen and slowly blink your eyes for 10 seconds."
+msgstr ""
+"Odvraťte pohled od obrazovky a pomalu mrkejte očima po dobu deseti sekund."
+
+#: src/Application.vala:158
+msgid "Eyes:"
+msgstr "Oči:"
+
+#: src/Application.vala:163
+msgid "Stretch your fingers"
+msgstr "Protáhněte prsty"
+
+#: src/Application.vala:164
+msgid "Spread out your palm wide, then close it into a fist. Repeat 5 times."
+msgstr "Roztáhněte dlaň doširoka a pak ji sevřete v pěst. Opakujte pětkrát."
+
+#: src/Application.vala:165
+msgid "Fingers:"
+msgstr "Prsty:"
+
+#: src/Application.vala:170
+msgid "Stretch your arms"
+msgstr "Protáhněte ruce"
+
+#: src/Application.vala:171
+msgid "Stretch your arms, and twist your wrists for 10 seconds."
+msgstr "Napněte paže a 10 sekund kruťte zápěstím."
+
+#: src/Application.vala:172
+msgid "Arms:"
+msgstr "Ruce:"
+
+#: src/Application.vala:177
+msgid "Stretch your legs"
+msgstr "Protáhněte nohy"
+
+#: src/Application.vala:178
+msgid "Stand up, twist each ankle, and bend each knee."
+msgstr "Postavte se, zakruťte každým kotníkem a pokrčte každé koleno."
+
+#: src/Application.vala:179
+msgid "Legs:"
+msgstr "Nohy:"
+
+#: src/Application.vala:184
+msgid "Turn your neck"
+msgstr "Otočte krkem"
+
+#: src/Application.vala:185
+msgid "Turn your head in all directions. Repeat 3 times."
+msgstr "Otáčejte hlavou všemi směry. Opakujte třikrát."
+
+#: src/Application.vala:186
+msgid "Neck:"
+msgstr "Krk:"
+
+#: src/Application.vala:191
+msgid "Watch your posture"
+msgstr "Sledujte své držení těla"
+
+#: src/Application.vala:192
+msgid "Make sure your back is straight."
+msgstr "Ujistěte se, že máte rovná záda."
+
+#: src/Application.vala:193
+msgid "Posture:"
+msgstr "Držení těla:"
+
+#: src/MainGrid.vala:59
+msgid "Reminders"
+msgstr "Připomínky"
+
+#: src/MainGrid.vala:74
+msgid "Decide how often Badger should remind you to relax these:"
+msgstr "Rozhodněte, jak často by vám měl Jezevec připomínat, abyste uvolnili:"
+
+#: src/MainGrid.vala:114
+msgid "1 min"
+msgstr "1 min"
+
+#: src/MainGrid.vala:116
+msgid "30 min"
+msgstr "30 min"
+
+#: src/MainGrid.vala:118
+msgid "1 hour"
+msgstr "1 hod"
+
+#: src/MainGrid.vala:149
+#, c-format
+msgid "%.0f min"
+msgstr "%.0f min"
+
+#: data/com.github.elfenware.badger.desktop.in:4
+#: data/com.github.elfenware.badger.appdata.xml.in:7
+msgid "Badger"
+msgstr "Jezevec"
```

Comment:
This is the app name that shows up in the AppCenter, so we leave it untranslated (see other languages).

Reply:
I'm used to translating the name from other apps as well. It looks more Czech then. But if you don't want to translate the name, I'll leave the original one.

Reply:
I don't have a strong opinion about this, but since we are keeping the English name in other languages, let's do the same in Czech for consistency.

Reply:
Done!